src/routes/trading-view/[chain]/tokens/index.svelte

```diff
@@ -1,86 +1,78 @@
-<script context="module" lang="typescript">
+<script context='module' lang='typescript'>
 	/*
-
 		Render listing of all available tokens
+	*/
+	import { buildBreadcrumbs } from '$lib/breadcrumb/builder';
+
+	export async function load({ url, params }) {
+		const pathTranslations = {
+			'trading-view': 'Trading data',
+			exchanges: 'Decentralised exchanges',
+			ethereum: 'Ethereum',
+			tokens: 'Tokens'
+		};
```

Comment:
This code can be simplified by getting ride of the `<script context='module'>` and `load` function – it is not needed for anything we are doing here.

Instead, you can pull this code directly into the main `<script>` context, import the `page` store (to access `url` and `params`) and just set `chain_slug` and `breadcrumbs` directly (instead of passing them as params). This should simplify the code quite a bit.

More info on the `page` store:
https://kit.svelte.dev/docs/modules#$app-stores-page
https://kit.svelte.dev/docs/types#sveltejs-kit-page

Import like this:
```svelte
import { page } from '$app/stores';
```

… and remember that it's a _store_, so use the special `$` syntax to de-reference its values – e.g.:
```svelte
const chain_slug = $page.params.chain;
```

Reply:
I don't see any changes to this.

Reply:
Pasting from Discord so this is captured here as well:

You should only `export` a variable from a Svelte component if it is being exposed as a prop (think of this as equivalent to a function's argument signature – don't include a var in the arg signature if it is never passed in as an arg).

For `page` components, you never load them the way you do other components (e.g., `<FooComponent bar={baz} />`, so the only time they should have any exported props is if you are passing something in via a module `load` function. Since we removed the (unneeded) `load` function, we can also remove the `export`-ed props (just make them local variables – in this case `const` because their state never changes w/in a given component instance).